index.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const fp = require('fastify-plugin')
-const lru = require('tiny-lru').lru
+const { LruObject } = require('toad-cache')
```

Comment:
Do we need an Lru here, or Fifo would suffice? Do we expect overflow and eviction?

Reply:
Hmm, seems like FIFO indeed looks more suitible, I don't see a need to modify the storage order depending on get calls

This cache only stores routes in a static way